### DISCUSSION TITLE
Brazil

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#501] [Galaxy Trivia] [answered]
FCFEEO
RCOCSE
NMOZAA
ACNVIALR
AOSBS AOVN

**Comment by peterbaccaro** [comment, marked as answer]
COFFEE :coffee:
SOCCER :soccer:
AMAZON :palm_tree:
CARNIVAL :dancer:
BOSSA NOVA :dancers:
